nntrainer/tensor/tensor.cpp

```diff
@@ -745,32 +745,59 @@ Tensor Tensor::getSharedDataTensor(const TensorDim dim_, unsigned int offset,
   return ret;
 }
 
-std::vector<Tensor> Tensor::split(unsigned num_size, int axis) {
+std::vector<Tensor> Tensor::split(std::vector<unsigned int> sizes, int axis) {
```

Comment:
and also, it is better to use ```size_t``` rather than ```unsigned int```

Reply:
Done